### DISCUSSION TITLE
Can i cutome the URL path of preview page?

### DISCUSSION THREAD

**Discussion by cheungseol** [storybookjs/storybook#24969] [Help]
### Summary

the version of storybook I used is 6.5.

When run `start-storybook -p 6006`, the default preview url is like this: `http://localhost:6006/?path=/story/pc-button--strong-button`.

Is there any way I can change the preview url , or use custom hash router, 
like `http://localhost:6006/#/?path=/custom/pc-button--strong-button`

**Comment by JReinhold** [comment]
It's currently not possible to modify the path of Storybook, but it is a commonly requested feature.

https://github.com/storybookjs/storybook/issues/25035
